test/common.js

```diff
@@ -18,7 +19,15 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
-exports.opensslCli = path.join(path.dirname(process.execPath), 'openssl-cli');
+if (process.config.variables.node_shared_openssl) {
+  // use external command
+  exports.opensslCli = 'openssl';
+} else {
+  // use build-in command
```

Comment:
@brendanashworth  Do you have a proper word? I think of others such as `bundled` or `included` command in iojs or to delete this comment.